packages/twenty-server/src/engine/metadata-modules/object-metadata/hooks/before-update-one-object.hook.ts

```diff
@@ -61,9 +80,48 @@ export class BeforeUpdateOneObject<T extends UpdateObjectPayload>
         };
       }
 
-      throw new BadRequestException(
-        'Only isActive field can be updated for standard objects',
-      );
+      if (
+        Object.keys(instance.update).every((key) =>
+          ['labelSingular', 'labelPlural', 'isLabelSyncedWithName'].includes(
+            key,
+          ),
+        )
+      ) {
+        if (
+          // eslint-disable-next-line no-prototype-builtins
```

Comment:
I think the pattern was already there but I'm curious why we had to disable it and if there's really no other way around, could you please explain? We try to avoid disabling lint rules

Reply:
I just copied the pattern above for getting a property.
I looked about using the `in` keyword but it doesn't have the exact same behavior (return properties including ones that aren't directly "owned" by the object, like prototype values). Not sure it would impact this specific object.

The ESLint warning recommends a much more verbose [solution](https://eslint.org/docs/latest/rules/no-prototype-builtins).

Reply:
Ok! Looks like the suggested alternative works indeed so let's replace it
`        if (
          Object.prototype.hasOwnProperty.call(
            instance.update,
            'isLabelSyncedWithName',
          ) &&
          instance.update.isLabelSyncedWithName !== undefined
        ) 
        ```